python/doc/source/api.rst

```diff
@@ -342,3 +342,15 @@ Apache Parquet
    read_schema
    write_metadata
    write_table
+
+.. currentmodule:: pyarrow
+
+Using with C extensions
+-----------------------
+
+.. autosummary::
+   :toctree: generated/
+
+   get_include
```

Comment:
Already made me a reminder to use them in `turbodbc` https://github.com/blue-yonder/turbodbc/issues/147. Much appreciated functionality.

Reply:
We should validate that they work as intended in both conda and pip based installs, I haven't had time to rigorously check